be/src/runtime/memory/mem_tracker_limiter.h

```diff
@@ -207,8 +251,13 @@ class MemTrackerLimiter final : public MemTracker {
 
     // Iterator into mem_tracker_limiter_pool for this object. Stored to have O(1) remove.
     std::list<MemTrackerLimiter*>::iterator _tracker_limiter_group_it;
+
+    taskgroup::TaskGroupPtr _task_group;
+    std::list<MemTrackerLimiter*>::iterator _tg_tracker_limiter_group_it;
 };
 
+using MemTrackerLimiterPtr = std::shared_ptr<MemTrackerLimiter>;
```

Comment:
doesn't seem to be used, can replace shared_ptr<>

Reply:
removed it